airbyte-integrations/connectors/source-zendesk-talk/integration_tests/spec.json

```diff
@@ -0,0 +1,49 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/zendesk-talk",
+  "changelogUrl": "https://docs.airbyte.io/integrations/sources/zendesk-talk",
+  "connectionSpecification": {
+    "title": "Zendesk Talk Spec",
+    "type": "object",
+    "properties": {
+      "subdomain": {
+        "title": "Subdomain",
+        "description": "The subdomain for your Zendesk Talk.",
+        "type": "string"
+      },
+      "access_token": {
+        "title": "Access Token",
+        "description": "The value of the API token generated. See the <a href=\"https://docs.airbyte.io/integrations/sources/zendesk-talk\">docs</a> for more information.",
+        "airbyte_secret": true,
+        "type": "string"
+      },
+      "email": {
+        "title": "Email",
+        "description": "The user email for your Zendesk account.",
+        "type": "string"
+      },
+      "start_date": {
+        "title": "Start Date",
```

Comment:
```suggestion
        "title": "Replication Start Date",
```

Reply:
just realized this file is for integration tests, pretend I made it on the "real " spec :P